mp/src/game/shared/neo/weapons/weapon_neobasecombatweapon.cpp

```diff
@@ -500,12 +500,16 @@ void CNEOBaseCombatWeapon::ProcessAnimationEvents()
 	else if (m_bLowered && m_bRoundBeingChambered)
 	{ // For bolt action weapons
 		m_bLowered = false;
-		next(ACT_VM_PULLBACK);
+		SendWeaponAnim(ACT_VM_PULLBACK);
+		m_flNextPrimaryAttack = max(gpGlobals->curtime + 1.2f, m_flNextPrimaryAttack);
+		m_flNextSecondaryAttack = m_flNextPrimaryAttack;
 	}
 
 	if (m_bLowered && gpGlobals->curtime > m_flNextPrimaryAttack)
 	{
-		next(ACT_VM_IDLE_LOWERED);
+		SetWeaponIdleTime(gpGlobals->curtime + 0.2);
+		m_flNextPrimaryAttack = max(gpGlobals->curtime + 0.2, m_flNextPrimaryAttack);
+		m_flNextSecondaryAttack = m_flNextPrimaryAttack;
 	}
```

Comment:
Since every path in the if-else chain above updates the `m_flNextPrimaryAttack` value as larger than `gpGlobals->curtime`, we could probably simplify this as:

```diff
diff --git a/mp/src/game/shared/neo/weapons/weapon_neobasecombatweapon.cpp b/mp/src/game/shared/neo/weapons/weapon_neobasecombatweapon.cpp
index b90b0cec..e9c21c2f 100644
--- a/mp/src/game/shared/neo/weapons/weapon_neobasecombatweapon.cpp
+++ b/mp/src/game/shared/neo/weapons/weapon_neobasecombatweapon.cpp
@@ -504,12 +504,10 @@ void CNEOBaseCombatWeapon::ProcessAnimationEvents()
                m_flNextPrimaryAttack = max(gpGlobals->curtime + 1.2f, m_flNextPrimaryAttack);
                m_flNextSecondaryAttack = m_flNextPrimaryAttack;
        }
-
-       if (m_bLowered && gpGlobals->curtime > m_flNextPrimaryAttack)
+       else if (m_bLowered)
        {
                SetWeaponIdleTime(gpGlobals->curtime + 0.2);
-               m_flNextPrimaryAttack = max(gpGlobals->curtime + 0.2, m_flNextPrimaryAttack);
-               m_flNextSecondaryAttack = m_flNextPrimaryAttack;
+               next(ACT_VM_IDLE_LOWERED);
        }
 }
```

Reply:
I'm purposefully avoiding sending the ACT_VM_IDLE_LOWERED animation here to prevent the jittering of the tachi and milso this pr is primarily fixing